tests/utility/v2/test_ca_response_builder.py

```diff
@@ -197,8 +198,10 @@ def test_get_premium_message_with_pvt_vul(self):
         obj.pvt_vul = 1
         obj.package = "django"
         obj.version = "1.1"
+        obj.severity = ['high']
         msg = obj.get_premium_message(1)
-        ideal_msg = "django - 1.1 has 1 security advisory with 1 exploitable vulnerabilities. "
+        ideal_msg = "django - 1.1 has 1 security advisory with 1 exploitable " \
+                    "vulnerability and high severity. No recommended version."
         self.assertEqual(msg, ideal_msg)
 
     def test_get_premium_message_with_public_vul(self):
```

Comment:
Add test case for '0' exploit and 'n' exploit.

Reply:
Done.